squad/frontend/urls.py

```diff
@@ -7,8 +7,14 @@
 from . import comparison
 from . import tests
 from . import ci
+from . import group_settings
+from . import project_settings
 from . import user_settings
-from squad.core.models import slug_pattern
+from squad.core.models import slug_pattern, group_slug_pattern
+
+
+group_and_user = (group_slug_pattern, slug_pattern)
```

Comment:
I might be missing something, but wouldn't `group_and_project` be a more appropriate name for this?

Reply:
yes! good catch

(updated)